src/components/ont/OntPoolEdit.vue

```diff
@@ -154,15 +163,16 @@ export default {
         this.busy = false
       })
     },
+
     // Allows users to upload a file to autopopulate the pool's selected libraries
-    async uploadFile(newFile) {
-      if (newFile === null) {
+    async uploadFile(evt) {
+      if (evt === null || evt.target.files === null || evt.target.files.length == 0) {
```

Comment:
Advisory:This could also be `if (!!evt?.target?.files?.length)` but that reverses the result so not sure if it works here.